src/components/UserMenu/UserMenu.types.ts

```diff
@@ -19,26 +19,26 @@ export type UserMenuProps = Omit<
 
 export type UserMenuI18N = Record<
   | 'myAssets'
-  | 'settings'
+  | 'account'
```

Comment:
Are you sure you want to remove `settings` and duplicate `account`?

Reply:
yeah, and thanks for the duplication catch.